OrbitGl/TimeGraph.cpp

```diff
@@ -646,14 +646,28 @@ std::vector<CallstackEvent> TimeGraph::SelectEvents(float a_WorldStart,
   sampling_profiler->SetGenerateSummary(a_TID == 0);
 
   for (const CallstackEvent& event : selected_callstack_events) {
-    if (Capture::GSamplingProfiler->GetCallStack(event.callstack_hash())) {
-      sampling_profiler->AddCallStack(CallstackEvent(event));
+    uint64_t callstack_id = event.callstack_hash();
+    if (Capture::capture_data_.GetCallstackData()->HasCallStack(callstack_id)) {
```

Comment:
introduce a variable for GetCallstackData()/GetSelectionCallstackData() - it will be easier to read code below

Reply:
I created a method `AddCallStackFromCallstackData` to fill SelectionCallstackData and introduced variables as well